doc/source/getting-started/experiment.rst

```diff
@@ -5,15 +5,27 @@ Define the platform
 -------------------
 
 To launch experiments on quantum hardware, users have first to define their platform.
-A platform is composed of a python file, with instruments information, and of a runcard file, with calibration parameters.
+To define a platform the user needs to provide a folder with the following structure:
+
+.. code-block:: bash
+
+    my_platform/
+        platform.py
+        parameters.yml
+        kernels.npz # (optional)
+
+where ``platform.py`` contains instruments information, ``parameters.yml``
+includes calibration parameters and ``kernels.npz`` is an optional
+file with additional calibration parameters.
```

Comment:
In principle, `parameters.yml` is also optional, as someone can define the qubit/channel objects hardcoded in the `create`. This is basically explained in the `lab.rst` tutorial.

However, dumping the platform after will automatically create parameters, so I am not sure whether we should advocate this kind of usage.